apps/poker-roll/src/app/startSession.jsx

```diff
@@ -0,0 +1,195 @@
+          type="text"
+          value={formState.game}
+          label="Game and Stakes"
+          onChange={handleChange}
+          variant="outlined"
+          size="small"
+          margin="normal"
+        />
+        <TextField
+          label="Casino"
+          variant="outlined"
+          name="casino"
+          type="text"
+          value={formState.casino}
+          onChange={handleChange}
+          size="small"
+          margin="normal"
+        />
+          <TextField 
+          label="Location"
+          variant="outlined"
+          name="location"
+          type="text"
+          value={formState.location}
+          onChange={handleChange}
+          size="small"
+          margin="normal"
+        />
+        <Button
+          type="submit"
+          size="small"
+          variant="contained"
+          color="primary"
+        >
+          Start CashGame
+        </Button>
+
+      </Stack>
+    )
+}
+
+function TournamentForm(){
+    const [formState, setFormState] = useState({
+        buyIn: "",
+        game: "",
+        casino: "",
+        location: "",
+      });
+
+      const handleChange = (event) => {
+        const { name, value } = event.target;
+    
+        setFormState({
+          ...formState,
+          [name]: value,
+        });
+      };
+      return(
+        <Stack
+          spacing={2}
+          direction="column"
+          justifyContent="center"
+          alignItems="center"
+        >
+          <h2>Enter New Tournament</h2>
+          <TextField
+            label="Tournamnet Buy In"
+            variant="outlined"
+            name="buyIn"
+            type="text"
+            value={formState.buyIn}
+            onChange={handleChange}
+            size="small"
+            margin="normal"
+          />
+          <TextField
+            name="game"
+            type="text"
+            value={formState.game}
+            label="Tournament Name"
+            onChange={handleChange}
+            variant="outlined"
+            size="small"
+            margin="normal"
+          />
+          <TextField
+            label="Casino"
+            variant="outlined"
+            name="casino"
+            type="text"
+            value={formState.casino}
+            onChange={handleChange}
+            size="small"
+            margin="normal"
+          />
+            <TextField 
+            label="Location"
+            variant="outlined"
+            name="location"
+            type="text"
+            value={formState.location}
+            onChange={handleChange}
+            size="small"
+            margin="normal"
+          />
+          <Button
+            type="submit"
+            size="small"
+            variant="contained"
+            color="primary"
+          >
+            Begin Tournament
+          </Button>
+          </Stack>
+      )
+}
+const NewSession = (props) => {
+    
+    const [alignment, setAlignment] = React.useState('cashgame');
```

Comment:
Go ahead and rename this variable something that will make more sense when you come back and look at this code a year from now. This is more like a `sessionType` or something. Always want to use relevant names so you can quickly get an idea of what the value is.